samples/v2/sample_test.py

```diff
@@ -37,6 +37,7 @@
 from kubernetes import client, config, utils
 import yaml
 from modelcar import modelcar
+from samples.v2 import pipeline_with_retry, pipeline_with_retry_all_args, pipeline_with_zero_retry
```

Comment:
I think this will fix your CI failure.
```suggestion
import pipeline_with_retry, pipeline_with_retry_all_args, pipeline_with_zero_retry
```

Reply:
Thanks for pointing that out, updated :+1: